Orange/widgets/unsupervised/owtsne.py

```diff
@@ -82,7 +80,7 @@ class OWtSNE(OWDataProjectionWidget):
     embedding_variables_names = ("t-SNE-x", "t-SNE-y")
 
     #: Runtime state
-    Running, Finished, Waiting = 1, 2, 3
+    Running, Finished, Waiting, Pending = 1, 2, 3, 4
```

Comment:
Maybe "Pending" isn't the greatest name for what this indicates. A better name would be "Paused". Which kind of begs the question of what "Waiting" means...